service/ui/src/controllers/learning-record/index.ts

```diff
@@ -110,13 +112,29 @@ async function getCourseRecord(user: any, course: any) {
 	const statements = response.data.statements
 	const state = getState(statements)
 	const result = getResult(statements)
+	const completionDate = getCompletionDate(statements)
 
 	return {
+        completionDate,
 		result,
 		state,
 	}
 }
 
+function getCompletionDate(statements: [any]) {
+	if (statements.length === 0) {
+		return null
+	}
+	const completed = statements.find(
+		statement =>
+			statement.verb.id === 'http://adlnet.gov/expapi/verbs/completed'
+	)
+	if (completed) {
+		return moment(completed.timestamp).format('DD/MM/YYYY')
```

Comment:
For future, could we use the builtin datetime formatting instead of `moment`? e.g.

```js
new Intl.DateTimeFormat('en-GB').format(date)
```